test/dev_test.pl

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env swipl
```

Comment:
Seems like this file is unnecessary after #32, no?

Reply:
removed